.github/ISSUE_TEMPLATE/icon_request.md

```diff
@@ -4,17 +4,23 @@ about: Request a new icon for SimpleIcons
 labels: new icon
 ---
 
-<!-- Before opening a new issue search for duplicate or closed issues -->
+<!--
+  We won't add non-brand icons or anything related to illegal services.
+  If in doubt, open an issue and we'll have a look.
 
+  Before opening a new issue please search for duplicate or closed
+  issues or PRs. If you find one for the brand your requesting then
+  leave a comment on it or add a reaction to the original post.
 
-<!-- When requesting a new icon we need information such as: -->
+  When requesting a new icon please provide the following information:
+-->
 **Name:**
 **Website:**
+**Alexa rank:**
+  <!-- The Alexa rank can be retrieved at https://www.alexa.com/siteinfo/
+       Please see our contributing guidelines at
+       https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+       for more details on how we assess a brand's popularity. -->
+
 **Official resources for icon and color:**
   <!-- for example media kits, brand guidelines, SVG files, ...) -->
```

Comment:
You didn't edit this but could you remove the closing parenthesis on this line now that we're editing this file anyway?

Reply:
Done :+1: I also replaced the ellipsis with "etc." for clarity.